### ISSUE TITLE
BUG - sev-3 - All - Some places in the app getting too much spacing

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8779]
## What happened?

Some pages on the mobile app are TOO MUCH spacing. Here's the 'how to change your legal name' large panel from personal info, as one example:

<img width="410" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNmt4QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1f1d37fb4bc4ee4031321b71b204a209b1a5d1e9/image.png" alt="image.png" />
<img width="410" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNnN4QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--29915f87c722a1547ca60f02de74679d94e5fd72/image.png" alt="image.png" />

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Desired behavior

Desired spacing is 40px between, but also the root cause would be good to know - custom spacing that interacted poorly with a component change? component modification elsewhere not realizing the impact here....?

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by brea11y** [comment]
After doing some research, we are definitely inconsistent in our paragraph spacing. 

Initially, I took some time to review the WCAG guidance and additional guidance from other sources for accessibility best practices in native mobile apps (including MCAG). All sources to recommend line-height at 1.5x the font size and 2x the font size for paragraph spacing, which is what was recommended during our WCAG updates. However, this does not seem to be what we are doing throughout the app (likely due to new components being implemented) and we are seeing areas (like in the example above), with inconsistent spacing and too much spacing between elements.

I did some [exploration in Figma](https://www.figma.com/design/BVfWDc8vinwx8kRc1LoqGl/%F0%9F%8F%96-Brea's-Sandbox?node-id=1296-883&t=9HWuO4tZoiKpTuFQ-0) and discovered that many of the elements on the screens shown in the example have as much as 67px between elements, with the average amount of spacing being 64px. This should be 40px and should not increase even if links are present.

I [included a side-by-side comparison](https://www.figma.com/design/BVfWDc8vinwx8kRc1LoqGl/%F0%9F%8F%96-Brea's-Sandbox?node-id=1297-4748&t=9HWuO4tZoiKpTuFQ-4) in the Figma file which shows what is in the app today vs. what we should actually have in the app. I also took a look at some of the other areas of the app (mostly focusing in on the health area where we have longer blocks of copy) and included [some examples of what the spacing should be](https://www.figma.com/design/BVfWDc8vinwx8kRc1LoqGl/%F0%9F%8F%96-Brea's-Sandbox?node-id=1297-5334&t=9HWuO4tZoiKpTuFQ-4). 

I've talked to @wavelaurenrussell quite a bit about the spacing inconsistencies and I know that this is something that she was looking to resolve in her recent work on appointments, but this is really an app-wide issue that we need to resolve. 

The purpose of the additional spacing after paragraphs is a way to break up large, flowing blocks of copy from other copy to improve legibility, but also as a way to help visually group content. While we don't always have to have exactly 40px between elements / below paragraphs, but it should look consistent visually, which is not what we are seeing in the app today.

**Comment by brea11y** [comment]
@ajsarkar28 – How would you like to move forward with this one now that we have identified that there is a spacing issue? Should I just close it out or should we expand and/or issue a new ticket to actually make adjustments to the spacing in the code?
